group_vars/location_k11/networks.yml

```diff
@@ -2,19 +2,19 @@
 ipv6_prefix: '2001:bf7:760:100::/56'
```

Comment:
pretty sure this has to change as well... IPv4 and IPv6 were registered together and should be again...

Reply:
Jep, forgot to include them in the request.